src/components/gui/gui.jsx

```diff
@@ -425,7 +425,6 @@ GUIComponent.defaultProps = {
     canEditTitle: false,
     canManageFiles: true,
     canRemix: false,
-    canSave: false,
```

Comment:
Confused why this is still moving out of GUI and into CloudManagerHOC. It's a prop for GUI, not anything to do with cloud variables, so it would make sense to me for it to live here.

Reply:
It does have to do with cloud variables, because it's used in the `shouldNotModifyCloudData` function.

In short, the idea there is that users who are not the project's owner should not be able to modify the code and then still be able to set cloud data (prevents the "high score hack"); whereas the project's owner should be able to still set cloud data.

Reply:
Do you understand why I think it's confusing for CloudManager to be setting the default for `canSave`?

Reply:
Resolving this per offline discussion